istio-proto/src/main/protobuf/proxy/v1/config/dest_policy.proto

```diff
@@ -59,7 +59,7 @@ message DestinationPolicy {
 // If tags are omitted, the policy applies for all versions of the
 // service. Policy CANNOT BE empty.
 // *Note:* Destination policies will be applied only if the corresponding
-// tagged instances are explicity routed to. In other words, for every
+// tagged instances are explicitly routed to. In other words, for every
```

Comment:
This is a direct mirror of https://github.com/istio/api/blob/6e481630954efcad10c0ab43244e8991a5a36bfc/proxy/v1/config/dest_policy.proto#L62 so we shouldn't edit this directly, even to fix typos

Reply:
Thank you for reminding me, @adleong.😸